docs/creators/configure-forem.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuring Forem
+---
+
+#Basic Site Configuration Guide for Forem
+
+##Overview
+
+As a Forem admin, one of the first steps of managing your site will be to tailor your content, branding and other important details based on your community goals. This is made simple by the configuration page which can be found at https://<span></span>your.forem.url/internal/config.
```

Comment:
The span is a trick so that markdown doesn't mark this as a link, because clicking on it will take the user nowhere, since it's a dummy one. No backticks required for this.